foreman/data_refinery_foreman/surveyor/external_source.py

```diff
@@ -51,7 +63,8 @@ def handle_batch(self, batch: Batch, key_values: BatchKeyValue = None):
         batch.survey_job = self.survey_job
         batch.source_type = self.source_type()
         batch.status = BatchStatuses.NEW.value
-        batch.internal_location = None
+        batch.internal_location = (self.source_type() + "/"
+                                   + batch.accession_code + "/")
```

Comment:
I agree with all of that :) At some point I'll probably change that root path to an environment variable.